Model/Import/Category.php

```diff
@@ -1,1505 +0,0 @@
-<?php
```

Comment:
We actively use the category importer in a 2.4 project and I can confirm it works. Please do not remove it! :)

Reply:
Okay, I reverted my change. I am happy that I removed it in a single commit, that made it much easier to revert.
I originally removed it because I saw https://github.com/firegento/FireGento_FastSimpleImport2/blob/develop/ResourceModel/Import/Category/Storage.php#L78: 
```
$collection = Mage::getResourceModel('catalog/category_collection')->addNameToResult();
/* @var $collection Mage_Catalog_Model_Resource_Eav_Mysql4_Category_Collection */
```
That class was referenced in the category import model, so I thought that all of it was non-functional.
I now found out that this class is not used, so I removed only this class.

I refactored the category import too now, but I didn't manage to run it successfully, I get `Please enter a correct entity model.` if I execute ` bin/magento fastsimpleimportdemo:category:import`. I am not sure if the demo is up to date, so I can't tell if my changes cause the failure.